### DISCUSSION TITLE
Determining which amount of lambdas is best

### DISCUSSION THREAD

**Discussion by hunterkwalt** [hahnlab/CAFE5#157] [Q&A] [answered]
I have run cafe with a uniform lambda, 2 lambdas, and three lambdas. Now, I would like to test if there are significant differences in the likelihood of these models. From the tutorial/other forums, I understand that I need to simulate data in order to do this. I have simulated data, but I am unsure how to generate a likelihood distribution from it. The tutorial seems to be unfinished in the simulation section. What is the best way to compare the likelihoods of the different models? I may just be missing something.

Thanks,
Hunter

**Comment by hahnlab-user** [comment, marked as answer]
See Discussion post #155
